src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -71,6 +73,8 @@ public FeedbackQuestionAttributes(String feedbackSessionName, String courseId, S
         this.showResponsesTo = showResponsesTo;
         this.showGiverNameTo = showGiverNameTo;
         this.showRecipientNameTo = showRecipientNameTo;
+        
+        removeIrrelevantVisibilityOptions();
```

Comment:
Will change to copy the lists. This constructor is actually not used, so maybe i will just remove it after confirming.